content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -306,6 +306,123 @@ For example, add the following configuration to your `datadog-values.yaml` file:
+
+#### Prerequisites
+
+- Kubernetes v1.20+
+- [`Helm`][1] for deploying the Datadog Operator
+- [`Kubectl` CLI][2] for installing the Datadog Agent
+
+#### Installation
+
+To enable single step instrumentation with the Datadog Operator:
+
+1. Install the [Datadog Operator][5] with Helm:
+   ```shell
+   helm repo add datadog https://helm.datadoghq.com
+   helm install my-datadog-operator datadog/datadog-operator
+   ```
+2. Create a Kubernetes secret to store your Datadog [API key][3]:
+   ```shell
+   kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY
+   ```
+2. Create `datadog-agent.yaml` with the spec of your Datadog Agent deployment configuration. The simplest configuration is as follows:
+   ```yaml
+   kind: DatadogAgent
+   apiVersion: datadoghq.com/v2alpha1
+   metadata:
+     name: datadog
+   spec:
+     global:
+       site: <DATADOG_SITE>
+       tags:
+         - env:<AGENT_ENV>
+       credentials:
+         apiSecret:
+           secretName: datadog-secret
+           keyName: api-key
+         appSecret:
+           secretName: datadog-secret
+           keyName: app-key
+   features:
+     apm:
+       instrumentation:
+         enabled: true  
+   ```
+   a. Replace `<DATADOG_SITE>` with your [Datadog site][6].  
+   b. Replace `<AGENT_ENV>` with the environment your Agent is installed on (for example, `env:staging`).
+
+3. Run the following command:
+   ```shell
+   kubectl apply -f /path/to/your/datadog-agent.yaml
+   ```
+4. Restart your applications.
+
+### Enabling or disabling instrumentation for namespaces
+
+You can choose to selectively instrument specific namespaces or choose to not instrument them.
+
+To enable instrumentation for specific namespaces, replace `enabled: true` with `enabledNamespaces` configuration in your `datadog-agent.yaml` file:
+{{< highlight yaml "hl_lines=4-6" >}}
+   features:
+     apm:
+       instrumentation:
+         enabledNamespaces: # Add namespaces to instrument
+           - namespace_1
+           - namespace_2
+ {{< /highlight >}}
+
+<div class="alert alert-info">The <code>enabled: true</code> option enables instrumentation for the entire cluster. You need to remove this to only enable instrumentation for specific namespaces.</a></div>
+
+To disable instrumentation for specific namespaces, add the `disabledNamespaces` configuration to your `datadog-agent.yaml` file:
+{{< highlight yaml "hl_lines=5-7" >}}
+   features:
+     apm:
+       instrumentation:
+         enabled: true
+         enabledNamespaces: # Add namespaces to instrument
+           - namespace_1
+           - namespace_2
+{{< /highlight >}}
+
+### Specifying tracing library versions
+
+You can optionally set specific tracing library versions to use. If you don't specify a version, it defaults to the latest version. To find the latest version for a library, go to **Releases** in the dd-trace-&lt;language&gt; GitHub repo. For example, [dd-trace-dotnet releases][15].
```

Comment:
telling people to go scan the repositories is such a poor experience 🙈 but we don't really have a better way today.

Reply:
We are updating this 😄